pkg/descheduler/evictions/evictions.go

```diff
@@ -517,7 +519,7 @@ func (pe *PodEvictor) EvictPod(ctx context.Context, pod *v1.Pod, opts EvictOptio
 		return err
 	}
 
-	ignore, err := pe.evictPod(ctx, pod)
+	ignore, err := pe.evictPod(ctx, pod, pe.policyGroupVersion, pe.gracePeriodSeconds)
```

Comment:
why is it being passed in as an argument, since it's already available on the PodEvictor receiver?

Reply:
done